kubejs/data/create_liquid_fuel/blaze_burner_fuel/tallowate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "fluid": "gtceu:tallowate",
```

Comment:
tallowate isn't an actual fluid -- it used to be fish_oil, but #2318 is changing it to just `tfc:tallow`